apps/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -498,6 +499,9 @@ export class RushConfiguration {
   // Lazily loaded when the projectsByName() getter is called.
   private _projectsByName: Map<string, RushConfigurationProject> | undefined;
 
+  private _commonVersionsConfigurations: Map<string, CommonVersionsConfiguration> | undefined;
+  private _implicitlyPreferredVersions: Map<string, Map<string, string>> | undefined;
```

Comment:
`Map<string, Map<string, string>>` is a bit vague about what all the key/values are.

Reply:
Maybe we could call it `_implicitlyPreferredVersionsByVariant`?

But there should at least be a comment saying something like:
```js
// A map from a variant name to an implicitlyPreferredVersions table.  The key is an empty
// string for the default variant.  The implicitlyPreferredVersions table is map from a 
// package name to a SemVer range.
```